src/library/coverartdelegate.cpp

```diff
@@ -108,17 +107,14 @@ void CoverArtDelegate::paintItem(QPainter *painter,
     info.hash = index.sibling(index.row(), m_iCoverHashColumn).data().toUInt();
     info.trackLocation = index.sibling(index.row(), m_iTrackLocationColumn).data().toString();
 
+    double scaleFactor = getDevicePixelRatioF(dynamic_cast<QWidget*>(parent()));
```

Comment:
A static cast should be sufficient here or am I wrong?

Reply:
You're right, fixed: 71e7ce4f05db4d595fa7113b58dbc5b19ee9fcec